runtime/compiler/arm/codegen/ARMRecompilationSnippet.cpp

```diff
@@ -64,103 +64,3 @@ uint32_t TR::ARMRecompilationSnippet::getLength(int32_t estimatedSnippetStart)
    {
    return  12;
    }
-/*
```

Comment:
This commented out block of code has long been deprecated.  It contained a reference to the `indexedTrampolineLookup` so I simply deleted the whole thing.